basic-examples/expressions/README.md

```diff
@@ -2,16 +2,78 @@
 
 ## Workflow Description
 
-This workflow demonstrates example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+This workflow demonstrates some example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/). Arcaflow expressions are a way to reference paths within the workflow or their fields, and/or manipulate the data.
+
+Arcaflow expressions were inspired by JSONPath but have diverged from the syntax. You can use expressions in a workflow YAML like this:
+
+```
+some_value: !expr $.your.expression.here
+```
+
+## Examples
+
+Basic path references are already used across the other example workflows. These include references in a step to an input from the schema:
+
+```
+steps:
+  example:
+    plugin:
+      deployment_type: image
+      src: quay.io/arcalot/arcaflow-plugin-example:0.5.0
+    input:
+      name:
+        _type: nickname

```

Comment:
`_type: nickname`? Is that really a thing? 😦

Reply:
Yes, that is how the schema for the example plugin is designed.